app/business/payments/merchant_registration/implementations/stripe/stripe_merchant_account_manager.rb

```diff
@@ -747,12 +747,21 @@ def self.handle_stripe_info_requirements(stripe_event_id, stripe_account, stripe
     end
 
     if stripe_account["payouts_enabled"]
-      user.update!(payouts_paused_internally: false)
+      user.update!(
```

Comment:
This shouldn't be done if the current payout pause was placed by an admin I think, should only be done if it was from Stripe.